File: mycuration/src/main/java/com/phicdy/mycuration/view/RegisterFilterView.java
package com.phicdy.mycuration.view;


public interface RegisterFilterView {
    String filterKeyword();
    String filterUrl();
    String filterTitle();
    void setFilterTargetRss(String rss);
    void handleEmptyTitle();
    void handleEmptyCondition();
    void handlePercentOnly();
    void finish();
    void showSaveSuccessToast();
    void showSaveErrorToast();
    void trackEdit();
    void trackRegister();
}

File: legacy/src/main/java/com/phicdy/mycuration/presentation/view/RegisterFilterView.kt
package com.phicdy.mycuration.presentation.view

interface RegisterFilterView {
    fun filterKeyword(): String
    fun filterUrl(): String
    fun filterTitle(): String
    fun setFilterTitle(title: String)
    fun setFilterTargetRss(rss: String)
    fun setMultipleFilterTargetRss()
    fun resetFilterTargetRss()
    fun setFilterUrl(url: String)
    fun setFilterKeyword(keyword: String)
    fun handleEmptyTitle()
    fun handleEmptyCondition()
    fun handlePercentOnly()
    fun finish()
    fun showSaveSuccessToast()
    fun showSaveErrorToast()
    fun trackEdit()
    fun trackRegister()
}
